### PR TITLE
adding implementation suggestions paper

### PR DESCRIPTION
**Pull request by nickdeveau94**
Adding in markdown version of our implementation paper

### CODE REVIEW

File: implementation_suggestions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# audit-AI: How we use it and what it does
+### pymetrics inc.
+### 25 November 2019
```
Comment (ljbaker) [inline implementation_suggestions.md]: Let's update this to 16 January 2020, given that it took a while to get this up

File: implementation_suggestions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Background: pymetrics models
+<p>Pymetrics models are built for specific roles within specific companies. To achieve this customization, we collect data from top-performing incumbents in the target role. We then compare incumbents to a baseline sample of the over 1 million candidates who have applied to jobs through pymetrics.  We also establish a special data set, which we call the debias set, which is sampled from a pool of 150,000 individuals who have voluntarily provided  basic demographic information such as sex, ethnicity or age. From there, a wide variety of algorithms might be tested to create an initial supervised machine learning model from the training data. The process itself is model agnostic. Multiple algorithms are fit in this process, and we are continuously testing new methods that might improve performance.  The goal of the algorithm is to find the trait profile that will most accurately and reliably separate the incumbent set from the baseline set. We create hundreds of possible models with slightly different model parameters to test and compare their performance before selecting the final model.</p>
```
Comment (ljbaker) [inline implementation_suggestions.md]: remove "supervised" to keep this more general

Comment (ljbaker) [inline implementation_suggestions.md]: Instead of "trait profile" put "features"

**Comment by nickdeveau94** [comment]
@ljbaker made the changes you suggesteed

**Comment by ljbaker** [comment]
Looks good, thanks!

**Pull request merged**